ilpycon/templates/homepage_en.html

```diff
@@ -62,26 +62,26 @@ <h3 class="text-center">Open Source</h3>
             </div>
 	    <div class="feature-columns row">
 	      <div class="col-md">
-		  <h3 class="text-center">Become a Speaker</h3>
+		  <h3 class="text-center">Presentations</h3>
 		  <p class="desc">
-		    The proposal submission is now closed. We are currently reviewing the
-		    submissions and preparing the agenda. Our thanks to all submitters!
+		    The <a href="/schedule/">schedule</a> is available for your reference.
 		  </p>
                 </div>
                 <div class="col-md">
-		  <h3 class="text-center">Register</h3>
+		  <h3 class="text-center">Registration</h3>
 		  <p class="desc">
-		    Tickets are waiting for you
-		    <a href="https://ti.to/pycon-israel/pycon-israel-2018">here</a>,
-		    but they won't wait forever...
+		    Registration is closed.
+		    If you have any questions, contact us directly at
+		    <a href="mailto:info@pycon.org.il">info@pycon.org.il</a>
 		  </p>
                 </div>
                 <div class="col-md">
-		  <h3 class="text-center">Become a Sponsor</h3>
+		  <h3 class="text-center">Getting there</h3>
 		  <p class="desc">
-		    We still have a few sponsorship slots open, if you want to show your
-		    support and make an impression on hundreds of enthusiastic pythonistas,
-		    <a href="mailto:info@pycon.org.il">contact us</a>.
+		    Wohl center, Max and Anna Webb st., Ramat Gan.
+		    See map in the <a href="/venue/">venue</a> page.
+		    There's also a ride-sharing group on
+		    <a href="https://chat.whatsapp.com/G7hV6onQD3V0LXJAqMvhmY" class="whatsapp-logo">WhatsApp</a>.
```

Comment:
AFAICT: Link does not work on a desktop. Should we add `(link works on phone only)` or something like that?

Also, should mention that `class="whatsapp-logo"` doesn't do much for now -- it depends on the image and css in #69